helpers/useReviewSectionRows.tsx

```diff
@@ -400,6 +417,14 @@ const useReviewSectionRows = <
                     </div>
                   ))}
                 </div>
+                <div>
+                  <a
+                    className="govuk-link"
+                    href={`${changeLink.as.pathname}?review=true`}
+                  >
+                    Change
+                  </a>
```

Comment:
This should be a `Link` tag (from `lbh-frontend-react`). This is so that it hooks in to the router properly so it will work offline.